### ISSUE TITLE
javascript not working production

### ISSUE THREAD

**Issue by sayanee** [webuildsg/webuild#239]
😱 😱 😨 😰 😓 😭 😂 😅  but! it's ok on [staging](https://staging-webuildsg.rhcloud.com/#check)

❗  Hence, search and video responsive size and title emojis are failing

![image](https://cloud.githubusercontent.com/assets/413879/15742713/360abf92-28f3-11e6-94db-72d804c72752.png)

form the deploy logs:

```
> webuild.sg@2.17.32 postinstall /var/lib/openshift/xxxxxxxxxxxxx/app-root/runtime/repo
> bower install || HOME=$HOME/app-root/runtime bower install && grunt build

/var/lib/openshift/xxxxxxxxxxxxx/app-root/runtime/repo/node_modules/bower/lib/node_modules/mkdirp/index.js:89
                    throw err0;
                    ^

Error: EACCES: permission denied, mkdir '/var/lib/openshift/xxxxxxxxxxxxx/.config'
    at Error (native)
    at Object.fs.mkdirSync (fs.js:842:18)
    at sync (/var/lib/openshift/xxxxxxxxxxxxx/app-root/runtime/repo/node_modules/bower/lib/node_modules/mkdirp/index.js:70:13)
    at Function.sync (/var/lib/openshift/xxxxxxxxxxxxx/app-root/runtime/repo/node_modules/bower/lib/node_modules/mkdirp/index.js:76:24)
    at Object.create.all.get (/var/lib/openshift/xxxxxxxxxxxxx/app-root/runtime/repo/node_modules/bower/lib/node_modules/configstore/index.js:39:13)
    at Object.Configstore (/var/lib/openshift/xxxxxxxxxxxxx/app-root/runtime/repo/node_modules/bower/lib/node_modules/configstore/index.js:28:44)
    at readCachedConfig (/var/lib/openshift/xxxxxxxxxxxxx/app-root/runtime/repo/node_modules/bower/lib/config.js:19:23)
    at defaultConfig (/var/lib/openshift/xxxxxxxxxxxxx/app-root/runtime/repo/node_modules/bower/lib/config.js:11:12)
    at Object.<anonymous> (/var/lib/openshift/xxxxxxxxxxxxx/app-root/runtime/repo/node_modules/bower/lib/index.js:16:32)
    at Module._compile (module.js:413:34)
[{
```

**Comment by subhransu** [comment]
I was about to report this. But saw you caught it already. 👍

**Comment by notthetup** [comment]
Fixed. `npm install` had failed to install bower properly. It should work now.

**Comment by sayanee** [comment]
Nope not working...

**Comment by sayanee** [comment]
Thanks @notthetup - working 🎉

**Comment by sayanee** [comment]
ok it failed again due to another push. Here's how to fix it every time (this is a temporary fix until #240):

1. `rhc ssh --app production`
- `cd app-root/repo`
- `grunt build`
- `exit`

**Comment by sayanee** [comment]
solved in #240
